test/functional/test-resources.js

```diff
@@ -126,6 +126,11 @@ describe('Resources', () => {
     expect(resources.calcTaskTimeout_(task_p0)).to.equal(0);
     expect(resources.calcTaskTimeout_(task_p1)).to.equal(0);
 
+    // Idle render penalty after first visible
+    resources.firstVisibleTime_ = 0;
```

Comment:
That's implicit in the "Empty pool" tests above. Would you like me to add the explicit `resources.firstVisibleTime_ = -1` there?